host/4/bullseye/amd64/java/java11-zulu/java11-composite.template

```diff
@@ -0,0 +1,34 @@
+ARG JAVA_VERSION=11u7
+
+FROM mcr.microsoft.com/java/jre-headless:${JAVA_VERSION}-zulu-debian10-with-tools as jre
```

Comment:
Looks not bullseye and using zulu. Is it expected?

Reply:
Agree with Tsuyoshi.  This image likely needs an update to Bullseye.  Zulu is expected though. Out of scope for your work Kaibo.  I will file an issue to update in the future.